src/app/destiny2/d2-definitions.test.ts

```diff
@@ -0,0 +1,13 @@
+import 'cross-fetch/polyfill';
+import { getTestDefinitions } from '../../testing/test-utils';
+import { D2ManifestDefinitions } from './d2-definitions';
+
+let defs: D2ManifestDefinitions;
+
+beforeAll(async () => {
+  defs = await getTestDefinitions();
```

Comment:
I wonder if we should run this from setupTests so that we can ensure the manifest is loaded for all tests and we can imply a file read will happen.

Reply:
That'd be an idea, you'd load it and throw it away but prime the cache.

Reply:
Yeah also combined with a GitHub action cache the tests would be pretty quick

Reply:
Another thought I had was load it as a global and the tests can just use it directly.